components/builder-web/app/actions/packages.ts

```diff
@@ -74,6 +76,39 @@ export function clearPackageVersions() {
   };
 }
 
+function setPackageCreatingFlag(payload) {
+  return {
+    type: SET_PACKAGE_CREATING_FLAG,
+    payload,
+  };
+}
+
+export function createEmptyPackage(body: object, token: string, callback: Function = (newPackage) => { }) {
+  return dispatch => {
+    dispatch(setPackageCreatingFlag(true));
+
+    new BuilderApiClient(token).createEmptyPackage(body).then((newPackage: any) => {
```

Comment:
Feeling like newPackage should be stricter than 'any' but leaving it open was causing an error for the name property on line 94 below.